cobalt/black_box_tests/black_box_tests.py

```diff
@@ -167,6 +189,31 @@ def LoadTests(launcher_params):
   return test_suite
 
 
+def LoadEvergreenEndToEndTests(launcher_params):
+  launcher = abstract_launcher.LauncherFactory(  # pylint: disable=unused-variable
+      launcher_params.platform,
+      'cobalt',
```

Comment:
Small readability nit: this gets a bit buried here, maybe pull it out into `EVERGREEN_TEST_BINARY='cobalt'` or such.

Reply:
Done